yams_py_tests/test_meta.py

```diff
@@ -0,0 +1,22 @@
+import re
+from pathlib import Path
+
+def test_correct_versions():
+    cs_version_regex = re.compile(r"<Version>(.*)<\/Version>")
+    python_version_regex = re.compile(r"version = (.*)")
+    cs_file = ""
+    python_file = ""
+
+    root_path = Path(__file__).parent.parent
+
+    with open(root_path.joinpath("YAMS-LIB", "YAMS-LIB.csproj")) as f:
+        cs_file = f.read()
+
+    with open(root_path.joinpath("setup.cfg")) as f:
```

Comment:
I'd use `importlib.metadata` to get the version, since it's what consumers of this package would have to do. Also ensures the installed package has the correct metadata.